sources/europe/fi/turku-orto-2018-true.geojson

```diff
@@ -0,0 +1,54 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "required": true,
+            "text": "\u00a9 Turun kaupunki",
+            "url": "https://www.turku.fi/turku-tieto/kartat-ja-paikkatieto"
+        },
+        "available_projections": [
+            "EPSG:3857",
+            "EPSG:4326"
+        ],
+        "best": true,
+        "country_code": "FI",
+        "description": "Ortophotos from the city of Turku",
+        "icon": "https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/Turku.vaakuna.svg/200px-Turku.vaakuna.svg.png",
+        "id": "turku-orto-2018-true",
+        "license_url": "https://dev.turku.fi/openstreetmap-lupa.pdf",
+        "privacy_policy_url": "https://www.turku.fi/turku-tieto/tietosuoja/kayttotiedot-ja-evasteet",
+        "max_zoom": 20,
+        "min_zoom": 4,
+        "name": "City of Turku ortophoto - 2018 True ortho",
```

Comment:
if this is from 2018, did you want to add the start_date and end_date properties as 2018? eg https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/oceania/au/act/ACTmapi-Imagery202001.geojson?short_path=9d34d6f#L18-L19

Reply:
Found and added the date